### DISCUSSION TITLE
Dual X Axis?

### DISCUSSION THREAD

**Discussion by Killajoedotcom** [lhndo/LH-Stinger#3] [General]
I saw mention of a dual X Axis mod. Are there any details on this? Is there anything I can do to help?

**Comment by lhndo** [comment]
Hey Killa, It should be pretty simple by mirroring the X mount assembly and adjusting the size of the X frame extrusions, or the ones of the X gantry and rail. 
Another idea discussed on the forums was an IDEX version with mirrored X gantry on a 2040 extrusion , something like this mockup: 
![image](https://github.com/lhndo/LH-Stinger/assets/42773937/88f833dd-2f22-4dcd-a5d9-01ba750f4845)

I think these are both worth looking at after the first release, but I feel it's a bit too early to tackle them now, and be able to properly test them. 
There's a bit of a backlog I would like to go though and harden the first full release. Things like minor design tweaks for accessibility, starting work on the documentation, setting up the klipper configs, working on finding more vendors for the custom parts, etc.

I'll reach to you on discord if you'd like to chat. I'd like to hear your ideas :)

**Reply by Killajoedotcom** [reply]
All sounds good to me. Agree a first release probably should be the priority. A discord chat would be great. 

Killa_Prints
